version/version.go

```diff
@@ -1,12 +1,14 @@
 package version
 
 var (
-	// TMCoreSemVer is the current version of Tendermint Core.
-	// It's the Semantic Version of the software.
-	TMCoreSemVer string
```

Comment:
Could other apps be relying on this? Would this be an API breaking change?

Reply:
ah I can change the name back for the backport, good call